packages/create-qwik/src/create-app-facade.ts

```diff
@@ -0,0 +1,18 @@
+import type { CreateAppOptions, CreateAppResult } from '../../qwik/src/cli/types';
+
+import { createApp } from './create-app';
+import { getPackageManager } from '../../qwik/src/cli/utils/utils';
+import { makeTemplateManager } from './helpers/templateManager';
+
+export async function createAppFacade(opts: CreateAppOptions): Promise<CreateAppResult> {
+  const pkgManager = getPackageManager();
+
+  const templateManager = await makeTemplateManager('app');
+
+  return await createApp({
```

Comment:
Hi @MrWaip
The CI is atm not happy with the CLI changes. looks like all apps on the e2e test return now `base` instead of the actual starter id. can you have a look at this? look like there is an error in `create-app.ts` in the response of the `createApp` fn. We always return `baseApp.id`. Just a thought after having a quick look. 

you can run `pnpm cli.validate` to get the errors. thx again.